### ISSUE TITLE
Issue, can't restore my nand saves from my old 3ds to my new 2ds ll

### ISSUE THREAD

**Issue by Dgb36** [FlagBrew/Checkpoint#472]
First, sorry if this repository is for bug reports only and I must post in the way provided here. But I really need help, when I try to restore my nand save data (streetpass mii plaza data, miis, playlog, friend list) from my n3ds xl to my n2ds ll it says the save data was restored successfully but nothing happened actually. And sometimes the app closes when I restore the saves. Is it possible to transfer the data this way, can I do something to fix the issue? Cant use the system transfer due to the consoles being different region. Thanks in advance for any help!

**Comment by BernardoGiordano** [comment]
Make sure you have initialized a save before restoring a backup with Checkpoint

**Comment by Dgb36** [comment]
> Make sure you have initialized a save before restoring a backup with Checkpoint
What is to initialize a save? Do I have to delete them or something? My main question is if it is really possible restore the nand save data in a console specially a region changed one, please tell me..

**Comment by Masamune3210** [comment]
You have to start the game at least once so the game can properly set up and initialize its save archive.

**Comment by BernardoGiordano** [comment]
You also need to save at least once.

**Comment by Dgb36** [comment]
> You have to start the game or app at least once so the game can properly set up and initialize its save archive.
Yes, I have started the titles and generated save files for them, otherwise they wouldnt show up in my Checkpoint account, could this be because I have to format the original system first pr I should do something in the Nand?

**Comment by Dgb36** [comment]
> You also need to save at least once.

Please i'd really like to get your response about this problem.

**Comment by Dgb36** [comment]
I could already restore the mii plaza data after I updated the game and got the dlc, after several attempts the playlog was restored too but not completely, only some titles data are showing, and for mii maker I scanned the qr codes of my miis, only friend list will be impossible i think.
